common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -56,14 +86,41 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
     val doc = <html>
       <body>
         <figure class={elementType}>
-          {iframe.toString}
+          {iframe}
         </figure>
       </body>
     </html>.toString()
-    val document: Document = Jsoup.parse(StringEscapeUtils.unescapeXml(doc))
+    val document: Document = parseTestData(doc)
+    clean(document)
+  }
+
+
+  private def cleanDocumentWithCommentEmbed(className: Option[String], src: Option[String], width: Option[String], height: Option[String], alt: Option[String]): Document = {
+
+    val classString = if(className.nonEmpty){s"""class=\"${className.get}\" """}else{""}
```

Comment:
Triple quote are quite useful to escape characters and write multi-lines strings. 
Therefore we don't need to escape `\"`
Also when using one line `if` the convention is not to use curly brackets. ex: `if(isX) "my string" else ""